mia/medium/famicom.cpp

```diff
@@ -132,7 +134,13 @@ auto Famicom::analyzeFDS(vector<u8>& data) -> string {
 auto Famicom::analyzeINES(vector<u8>& data) -> string {
   string hash = Hash::SHA256({data.data() + 16, data.size() - 16}).digest();
   string manifest = Medium::manifestDatabase(hash);
-  if(manifest) return manifest;
+  if(manifest) {
+    manifest += "    memory\n";
```

Comment:
This will add an entry to the end, obviously. Order doesn't matter to the code that uses the manifest. I don't think this is a big deal, but I wanted to call it out.